nuggetsapp/models.py

```diff
@@ -1,6 +1,9 @@
 from django.db import models
 from django.utils import timezone
 from django.contrib.auth.models import User
+import datetime
+from datetime import timedelta
+
```

Comment:
Yeah, would also be good to either rename `Q` or explicitly comment what it does (prefer rename).  Right now, not clear what it does.